src/ResourceManagement/HDInsightJob/HDInsightJob/packages.config

```diff
@@ -3,10 +3,13 @@
   <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.DataLake.Store" version="1.0.3" targetFramework="net45" />
   <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.2" targetFramework="net45" />
```

Comment:
There is no reuse of credentials between HDInsight client and DataLake client. Both of them uses different credentials. HDinsight uses Hyak based username password and run jobs on HDInsight cluster. Once job run is complete, customers creates Datalake client object probably using TokenCredentials. HDInsight .Net SDK accepts DataLake client object and uses it to retrieve job output.